tools/emr/README.md

```diff
@@ -24,10 +24,12 @@ In AWS IAM, add the following roles with **Create Role** | **AWS service** | **E
 
 ## Install the required libraries
 
-1. From the repository root, run:
+Make sure you use pip 21.1 or newer.
+
+1. From `tools`, run:
 
 ```
-pip install -r tools/requirements.txt
+pip install -e .
```

Comment:
The command
```
pip install --user -e .
```
fails on my machine (Fedora 34, pip 21.2.4).
It seems this is caused by this issue: https://github.com/pypa/pip/issues/7953